SA/design_concept/content/design/design_evaluation_decision.typ

```diff
@@ -0,0 +1,15 @@
+= Decision
```

Comment:
I'm not sure that we need an actual "decision" section. With iterations the final design proposal is just the result of the last iteration (maybe we can descibe that a bit clearer at the start of the chapter). The remaining open questions can be answered in the next section, instead of saying that we don't answer them yet.

Reply:
I feel like we have to say something to end the evaluation phase. I reworked it to `Conclusion`, does that work for you?

Reply:
Yes that works. I've made a suggestion based on what you wrote.